tests/ltp/README.md

```diff
@@ -56,7 +56,7 @@ FAILING
 | /ltp/testcases/kernel/syscalls/alarm/alarm07 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/bind/bind01 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/bind/bind02 | 0 | 0 | 0 |
-| /ltp/testcases/kernel/syscalls/bind/bind03 | 1 | 0 | 0 |
+| /ltp/testcases/kernel/syscalls/bind/bind03 | 0 | 0 | 0 |
```

Comment:
Did bind03 test on ext2fs pass in the past?

Reply:
Yes it did, there was another PR to remove it

Reply:
@salsal97, is the root cause of the failure known?

Reply:
Hi Bo, a couple of them were taken out at the same time, tracked here https://github.com/deislabs/mystikos/issues/599

Reply:
And the PR details are here https://github.com/deislabs/mystikos/issues/562